release-deploy/action.yml

```diff
@@ -32,8 +34,14 @@ runs:
         GYG_REGION: ${{ inputs.gyg_region }}
         RELEASE_VERSION: ${{ inputs.version }}
       run: |
-        yarn release:diff || true
-        yarn release:deploy
+        RUN_CMD="${{ inputs.run_cmd }}"
+        if [[ ${RUN_CMD+exists} == "" ]];
+        then
+          yarn release:diff || true
+          yarn release:deploy
+        else
+          ${{ inputs.run_cmd }}
```

Comment:
This feels like a bit of a security hole. Any old command could be passed in here.

That being said, I don't know if I know the solution. Maybe we could just pass in two yarn commands. eg.
```
yarn_target_1=release:diffXXX
yarn_target_2=release:deployXXX
```
and then this action could be:
```
yarn ${{ inputs.yarn_target_1 }} || true
yarn ${{ inputs.yarn_target_2 }}
```
Open to other options though

Reply:
@edwardaux Please look at the latest changes

Release is now back to using the default - features can create a `release:build` that references the other `release:build:*` - I will note this in the documentation

Deploy now takes the deploy name suffix. There is a naming convention required for this which I will also add to the documentation